docs/GettingStarted.md

```diff
@@ -358,3 +358,10 @@ and checked the Development checkbox, installation was completed for you.
 Otherwise, after installing the nuget package, as a one-time operation, you will
 currently need to run the `export_program_info.exe` tool to complete the install. This
 tool can be found in your project's `packages\eBPF-for-Windows\build\native\bin` directory.
+
+If you are working on the EbpfCore or the NetEbpfExt drivers, you may find the WinDbg command ```.kdfiles``` to be useful. This
```

Comment:
Does this also apply when using the VS [Debugger Immediate Window](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/entering-debugger-commands-in-visual-studio)?  If not, then update the first part of the sentence, as in:

```suggestion
If you are using WinDbg to work on the EbpfCore or the NetEbpfExt drivers, you may find the WinDbg command ```.kdfiles``` to be useful. This
```

If kdfiles does in fact work in the VS debugger immediate window, then update latter half of the sentence to not be limited to WinDbg.

Reply:
Driver debugging using VS requires the installation of the WDK on the test device [(for VS intergration).](https://learn.microsoft.com/en-us/windows-hardware/drivers/gettingstarted/provision-a-target-computer-wdk-8-1).  Unfortunately, 'Note' @ the [WDK install page](https://learn.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk) mentions that driver debugging under VS is currently non-functional and instead recommends using WinDbg instead.

I've updated this verbiage accordingly.